apps/mudita-center-e2e/src/page-objects/tabs.page.ts

```diff
@@ -23,6 +23,12 @@ class NavigationTabs extends Page {
     await this.overviewTab.click()
   }
 
+  public get overviewKompaktTab(): ChainablePromiseElement<
+    Promise<WebdriverIO.Element>
```

Comment:
We can remove chainable promise events - thats a leftover from 2022 when Pure was using this Page object
![Uploading Screenshot 2024-06-26 at 08.42.43.png…]()

Reply:
fixed